charts/argo-cd/values.yaml

```diff
@@ -915,14 +932,14 @@ server:
   ## Certificate configuration
   certificate:
     # -- Enables a certificate manager certificate
-    enabled: false
+    enabled: true
     # -- Certificate manager domain
     domain: argocd.example.com
```

Comment:
Is this domain correct? Is this for the ingress? Then it should be the dns name, no?

Reply:
I need to remove that field now, we reference domain elsewhere, I will go back and verify I didn't miss anything else.